users/drashna/.gitignore

```diff
@@ -0,0 +1,3 @@
+secrets.c
+secrets.h
+drashna_song_list.h
```

Comment:
:eyes:

Reply:
If you're wondering.... it's a better way to manage the files I don't want git including, rather than using the annoying `.git/info/exclude` file....   and this allows me to maintain a local list of songs that may be copyrighted, and that we don't want in the repo.

Reply:
Oh, I know what the file does. I was more wondering what songs you have in there that you aren't sharing with the rest of us.

Reply:
Probably: Baby got back